dpctl/__init__.pxd

```diff
@@ -29,4 +29,4 @@
 
 from dpctl._sycl_core cimport *
 from dpctl._memory import *
-
+from .dparray import *
```

Comment:
Why is this needed in the `__init__.pxd`? It does not really belong here.

Reply:
Maybe this is how he got dparray to be visible if the user does dpctl.dparray.

Reply:
I have removed it. Works the same.

Reply:
The testing code is referencing dpctl.dparray and it is working?  If so then perhaps we don't need this line but I'm still curious how this is working now when I had so much trouble getting it to work.